bftengine/src/bftengine/ReplicasInfo.cpp

```diff
@@ -143,6 +143,13 @@ ReplicasInfo::ReplicasInfo(const ReplicaConfig& config,
         }
         if (start != end) LOG_INFO(GL, "Principal ids in _idsOfInternalClients: " << start << " to " << end - 1);
         return ret;
+      }()},
+      _idsOfOperators{[&config]() {
```

Comment:
Let's keep also the order of initialisations correct. _idsOfOperators should appear between _idsOfClientServices and _idsOfInternalClients. Please move the log 
if (config.operatorEnabled_) LOG_INFO(GL, "Operator id: " << end); from _idsOfClientServices to here.